### PR TITLE
CIV-12870 Service PR - Docmosis template for Defendant LiP PIN in Post Letter - Judgment by admission

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-12870

### Change description ###
Service PR - Docmosis template for Defendant LiP PIN in Post Letter - Judgment by admission

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/service/FeatureToggleService.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public boolean isSdoR2Enabled() {
     }
 
     public boolean isJudgmentOnlineLive() {
-        return featureToggleApi.isFeatureEnabled("isJudgmentOnlineLive");
+        return true; //TODO: Revert this
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/FeatureToggleService.java:102]: Temporarily enabled this toggle for testing. To be reverted

File: build.gradle [hunk 2]
```diff
@@ -337,7 +337,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.88'
+  implementation 'com.github.hmcts:civil-commons:v1.0.88_CIV_12870'
```
Comment (sjamihmcts) [inline build.gradle:340]: To be updated once civil-commons PR is merged.

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/JudgmentByAdmissionPinInPostLiPDefendant1LetterHandler.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse sendJudgmentByAdmissionLetterToLiPDefendant1(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        if (isRespondent1Lip(caseData) && caseData.getRespondent1() != null && caseData.getRespondent1().getPartyEmail() != null) {
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/JudgmentByAdmissionPinInPostLiPDefendant1LetterHandler.java:51]: Re-check the partyEmail condition

File: src/test/java/uk/gov/hmcts/reform/civil/service/FeatureToggleServiceTest.java [hunk 4]
```diff
@@ -200,14 +200,14 @@ void shouldReturnCorrectValue_whenIsSdoR2Enabled(Boolean toggleStat) {
         assertThat(featureToggleService.isSdoR2Enabled()).isEqualTo(toggleStat);
     }
 
-    @ParameterizedTest
```
Comment (sjamihmcts) [inline src/test/java/uk/gov/hmcts/reform/civil/service/FeatureToggleServiceTest.java:203]: To be reverted.

File: src/main/java/uk/gov/hmcts/reform/civil/service/FeatureToggleService.java (hunk 1 above)
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/service/FeatureToggleService.java:102]: Done

File: src/test/java/uk/gov/hmcts/reform/civil/service/FeatureToggleServiceTest.java (hunk 4 above)
Comment (sjamihmcts) [reply src/test/java/uk/gov/hmcts/reform/civil/service/FeatureToggleServiceTest.java]: Done

**Review by hmcts-version1-ignacio — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/DocmosisTemplates.java [hunk 5]
```diff
@@ -45,6 +45,7 @@ public enum DocmosisTemplates {
     HEARING_OTHER("CV-UNS-HNO-ENG-01196.docx", "hearing_other_claim_%s.pdf"),
     PIN_IN_THE_POST_LETTER("CV-CMC-LET-ENG-PIP0001.docx", "PIP_letter.pdf"),
     SET_ASIDE_JUDGMENT_IN_ERROR_LIP_DEFENDANT_LETTER("CV-CMC-LET-ENG-LIP-JO0001.docx", "set_aside_letter.pdf"),
+    JUDGMENT_BY_ADMISSION_PIN_LIP_DEFENDANT_LETTER("CV-CMC-LET-ENG-LIP-JO0005.docx", "judgment_by_admission_letter.pdf"),
```
Comment (Azam-Hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/DocmosisTemplates.java:48]: Is this is shown anywhere or it has no impact when document shown to user

Comment (Azam-Hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/DocmosisTemplates.java:48]: can we use this "PIN_IN_THE_POST_LETTER("CV-CMC-LET-ENG-PIP0001.docx", "PIP_letter.pdf")," or its template is different from the letter we are generating? It seems to be something generic.

**Review by German-A-Amoros — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/DocmosisTemplates.java (hunk 5 above)
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/DocmosisTemplates.java]: Have updated the name as per your suggestion.

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by German-A-Amoros — DISMISSED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Pull request merged**